test/utils/update/trust.model.fetch.spec.ts

```diff
@@ -791,6 +791,71 @@ describe("Test fetching and mapping of Trust data", () => {
     expect(trustMock.HISTORICAL_BO).toHaveLength(2);
   });
 
+  test("should not map trustees into trust when trustees are ceased", () => {
+    trustMock.INDIVIDUALS = [];
+    trustMock.CORPORATES = [];
+    trustMock.HISTORICAL_BO = [];
+
+    const ceasedIndividualTrusteeData = {
+      hashedTrusteeId: "1",
```

Comment:
i'd give them separate hashedTrusteeId, just to make it obvious they should not be the same

Reply:
Unique ids now assigned.